zstacklib/zstacklib/network/ovs/controller.py

```
@@ -0,0 +1,548 @@
+        _validate_name(port_name, "port name")
+        try:
+            if port_type:
+                _validate_name(port_type, "port type")
+                cmd = CTL_BIN + f'add-port {br_name} {port_name} -- set Interface {port_name} type={port_type} '
+                for opt in options:
+                    _validate_name(opt.split('=')[0] if '=' in opt else opt, "option")
+                    cmd += f'options:{opt} '
+            else:
+                cmd = CTL_BIN + f'add-port {br_name} {port_name}'
+            shell.call(cmd)
+        except Exception as err:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)